src/blake.rs

```diff
@@ -56,17 +56,19 @@ macro_rules! implement_blake {
             fn blake<T: AsRef<[u8]>>(s: T) -> Self {
                 let input = s.as_ref();
                 let mut result = Self::default();
-                let mut hasher = Blake2b::new(result.len());
+                let mut hasher = VarBlake2b::new(result.len()).unwrap();
```

Comment:
How about it?
```rust
macro_rules! implement_blake {
    ($self:ident, $size:expr) => {
        impl Blake for $self {
            fn blake<T: AsRef<[u8]>>(s: T) -> Self {
                Self::blake_with_key(s, &[])
            }
            fn blake_with_key<T: AsRef<[u8]>>(s: T, key: &[u8]) -> Self {
                let input = s.as_ref();
                let mut hasher = VarBlake2b::new_keyed(&key, $size);
                hasher.input(input);
                let mut result: [u8; $size] = [0; $size];
                result.copy_from_slice(&hasher.vec_result());
                Self(result)
            }
        }
    };
}

implement_blake!(H128, 16);
implement_blake!(H160, 20);
implement_blake!(H256, 32);
implement_blake!(H512, 64);
```
It will bypass the clippy error.

Reply:
Sounds good! To solve this error, I replaced fixed-hash@0.2.5 to @0.2.2, but this methods looks better. I'll modify it. Thank you.